src/lang/en-US.ts

```diff
@@ -612,11 +615,11 @@ const CaptureCreate: ResolvedIntlConfig['messages'] = {
     'captureCreate.config.source.homepage': `Home`,
     'captureCreate.save.failed': `Capture creation failed. See below for details:`,
     'captureCreate.editor.default': `Before you can edit the capture specification, you must fill out the Connection Configuration section and click "${CTAs['cta.generateCatalog.capture']}." `,
-    'captureCreate.finalReview.instructions': `The following Flow specification was generated from the details you provided. To make changes, you can enter new values in the form above and click "${CTAs['cta.generateCatalog.capture']}" again. You can also edit the specification file directly. Click "${CTAs['cta.saveEntity']}" to proceed.`,
+    'captureCreate.finalReview.instructions': `The following Flow specification was generated from the details you provided. To make changes, you can enter new values in the form above and click "${CTAs['cta.generateCatalog.capture']}" again. You can also edit the specification file directly below. Click "${CTAs['cta.saveEntity']}" to proceed.`,
 
     'captureCreate.collections.heading': `Output Collections`,
     'captureCreate.collectionSelector.heading': `Collection Selector`,
-    'captureCreate.collectionSelector.instructions': `The collections bound to your capture. To make changes, you can enter new values in the this section of the form or edit the YAML file shown in the ${EntityCreate['entityCreate.catalogEditor.heading']} section below.`,
+    'captureCreate.collectionSelector.instructions': `The collections bound to your capture. To update the configuration please update the fields under the Config tab. To update the schema click on Edit under the Specification tab.`,
```

Comment:
@oliviamiannone - this change is made several times. Basically removing a reference to YAML that never showed up. Also, giving more specifics on how to edit the config/schema now that there is more than just one big spec editor.

Reply:
Definitely an improvement! Few grammar nits:
* needs commas after "To update the configuration," and "To update the schema,"
* "on" in "click on Edit" can be removed

Reply:
Updated